src/commands/introduction/index.ts

```diff
@@ -138,11 +138,11 @@ export default class Intro extends BaseCommand<typeof Intro> {
     }
     helpText['exit'] = {id: 'exit', value: 'exit', text: 'Goodbye.'}
 
-    this.log('\n' + ux.colorize('blue', introText))
+    this.log('\n' + ux.colorize('blueBright', introText))
     this.log(connectionsGraphModelBasic)
 
-    this.log('\n' + ux.colorize('blue', introText2))
-    this.log('\n' + ux.colorize('blue', connectionsIntegrationsIntroText))
+    this.log('\n' + ux.colorize('blueBright', introText2))
```

Comment:
q: what does this look like on Powershell's blue background?

Reply:
has to be better than the previous dark blue that was too difficult to read. gonna try to install powershell to test this out, but can't be worse for sure.

Reply:
![image](https://github.com/user-attachments/assets/45563600-a527-4776-96d6-6d1258b3f9db)

I think this is definitely an improvement.